src/consumerIncentives/ConsumerIncentivesHomeScreen.test.tsx

```diff
@@ -133,6 +148,30 @@ describe('ConsumerIncentivesHomeScreen', () => {
     )
 
     expectVisibleMainComponents(queryByTestId, 'SuperchargeInstructions')
+    expect(getByTestId('SuperchargeInstructions/ConnectNumber')).toHaveTextContent(
```

Comment:
unfortunately `getByText` looks for an exact match, whereas the mock i've defined outputs the key + the react nodes